Compilation error under windows

```nushell
cargo build --release                                                                                                              11/18/2024 11:29:02 AM
   Compiling serde v1.0.193
   Compiling home-dir v0.1.0
   Compiling tokio-stream v0.1.14
error[E0432]: unresolved import `nix::unistd`
  --> D:\scoop\persist\rustup\.cargo\registry\src\index.crates.io-6f17d22bba15001f\home-dir-0.1.0\src\lib.rs:14:10
   |
14 | use nix::unistd::{Uid, User};
   |          ^^^^^^ could not find `unistd` in `nix`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `home-dir` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

> cargo version 
cargo 1.82.0 (8f40fc59f 2024-08-21)
> build_target
x86_64-pc-windows-msvc

--- a/lsp-copilot.el
+++ b/lsp-copilot.el
@@ -177,9 +177,6 @@ the buffer when it becomes large."
 (defvar-local lsp-copilot--change-idle-timer nil
   "Idle timer for didChange signals.")
 
-(defvar lsp-copilot--xref-callback nil
-  "XREF callback.")
-
 (defvar-local lsp-copilot--completion-trigger-characters nil
   "Completion trigger characters.")
 
@@ -691,24 +688,20 @@ LSP server result."
   (list (propertize (or (thing-at-point 'symbol) "")
                     'identifier-at-point t)))
 
-(cl-defmethod xref-backend-definitions ((_backend (eql xref-lsp-copilot)) _identifier callback)
+(cl-defmethod xref-backend-definitions ((_backend (eql xref-lsp-copilot)) _identifier)
   (save-excursion
-    (setq lsp-copilot--xref-callback callback)
     (lsp-copilot-find-definition)))
 
-(cl-defmethod xref-backend-references ((_backend (eql xref-lsp-copilot)) _identifier callback)
+(cl-defmethod xref-backend-references ((_backend (eql xref-lsp-copilot)) _identifier)
   (save-excursion
-    (setq lsp-copilot--xref-callback callback)
     (lsp-copilot-find-references)))
 
-(cl-defmethod xref-backend-implementations ((_backend (eql xref-lsp-copilot)) _identifier callback)
+(cl-defmethod xref-backend-implementations ((_backend (eql xref-lsp-copilot)) _identifier)
   (save-excursion
-    (setq lsp-copilot--xref-callback callback)
     (lsp-copilot-find-implementations)))
 
-(cl-defmethod xref-backend-type-definitions ((_backend (eql xref-lsp-copilot)) _identifier callback)
+(cl-defmethod xref-backend-type-definitions ((_backend (eql xref-lsp-copilot)) _identifier)
   (save-excursion
-    (setq lsp-copilot--xref-callback callback)
     (lsp-copilot-find-type-definition)))
 
 

